server/db/seed/seed.js

```diff
@@ -0,0 +1,159 @@
+var fs = require('fs');
+
+var knex = require('knex')({
+  client: 'mysql',
+  version: '5.7',
+  connection: {
+    host : '127.0.0.1',
+    port: 3306,
+    user : 'root',
+    password : 'password',
+    multipleStatements: true
+  }
+});
+
+var sql = fs.readFileSync('./server/db/seed/database_base_data.sql').toString();
+console.log('Loaded SQL insert statements, starting drop of database if exists');
+console.log('This may take a bit');
+
+knex.raw('DROP DATABASE IF EXISTS scorecard')
+    .then(() => {
+      console.log('Dropped scroecard database now (re)creating');
+      return knex.raw('CREATE DATABASE scorecard')
+    })
+    .then(() => {
+      console.log('As the connection to the database was killed destroy and recreate knex to prevent errors');
+      knex.destroy();
+      knex = require('knex')({
+        client: 'mysql',
+        version: '5.7',
+        connection: {
+          host : '127.0.0.1',
+          port: 3306,
+          user : 'root',
+          password : 'password',
+          database : 'scorecard',
+          multipleStatements: true
+        }
+      });
+      console.log('Now create the tables and insert the base movies and scraped critic reviews for blank panther.');
+      return;
+    })
+    .then(() => knex.raw(sql))
+    .then(() => {
+      console.log('Old Database Dropped and Recreated with Base Critics');
+      console.log('Starting to generate Fake Reviews for each of the movies');
+
+      populate_reviews();
+    });
+
+
+
+
+function populate_reviews ()  {
+  var critic_reviews = [];
+
+  knex('movies').select(['id'])
+  .then(function(results) {
+    for (var i=0; i<results.length;i++) {
+      //prevent reviews for black panther
+      if (results[i].id !==284054 ) {
+        let movie_id = results[i].id;
+        //create a starting score that will ultimately decide whether the movie is rotten, fresh or certified fresh
+        let startRandom = (2 + Math.floor(Math.random() * 5));

```

Comment:
It's not clear to me where the definition (2+someNumber*5) comes from. It would be good to make a comment below.